client/src/pages/AdminPage/AdminBlog/index.js

```diff
@@ -0,0 +1,385 @@
+        {blogPost && ('Edit')}
+        {!blogPost && ('Create')}
+        {' '}
+        blog post
+      </Modal.Title>
+    </Modal.Header>
+    <Modal.Body>
+      <Form>
+        <Form.Group controlId="formBasicAuthor">
+          <Form.Label>Author</Form.Label>
+          <Form.Control
+            type="text"
+            placeholder="Author"
+            defaultValue={blogPost && blogPost.author}
+            disabled
+          />
+        </Form.Group>
+        <Form.Group controlId="formBasicTitle">
+          <Form.Label>Title</Form.Label>
+          <Form.Control
+            type="text"
+            name="title"
+            placeholder="Title"
+            defaultValue={
+              blogPost && blogPost.title
+            }
+            onChange={handleChange}
+          />
+        </Form.Group>
+        <Form.Group controlId="formBasicSubtitle">
+          <Form.Label>Subtitle</Form.Label>
+          <Form.Control
+            type="text"
+            name="subtitle"
+            placeholder="Subtitle"
+            defaultValue={
+              blogPost && blogPost.subtitle
+            }
+            onChange={handleChange}
+          />
+        </Form.Group>
+        <Form.Group controlId="formBasicContent">
+          <Form.Label>Content</Form.Label>
+          <Form.Control
+            as="textarea"
+            rows="10"
+            type="text"
+            name="content"
+            placeholder="Content"
+            defaultValue={
+              blogPost && blogPost.content
+            }
+            onChange={handleChange}
+          />
+        </Form.Group>
+      </Form>
+      {error && (<p className="text-danger">{error}</p>)}
+    </Modal.Body>
+    <Modal.Footer>
+      {blogPost && (
+        <Button
+          variant="success"
+          onClick={() => handleSave(blogPost._id)}
+        >
+          Save
+        </Button>
+      )}
+      {!blogPost && (
+        <Button
+          variant="success"
+          onClick={handleCreate}
+        >
+          Create
+        </Button>
+      )}
+      <Button variant="danger" onClick={closeModal}>Close</Button>
+    </Modal.Footer>
+  </Modal>
+)
+
+const ConfirmModal = ({
+  showConfirmModal, closeConfirmModal, handleDelete, postId
+}) => (
+  <Modal
+    show={showConfirmModal}
+    size="lg"
+    aria-labelledby="contained-modal-title-vcenter"
+    centered
+  >
+    <Modal.Header>
+      <Modal.Title id="contained-modal-title-vcenter">
+        Confrim your action
+      </Modal.Title>
+    </Modal.Header>
+    <Modal.Body>
+      <h4>Are you sure?</h4>
+      <p>
+        Once you will delete this blog post it will be removed forever. You
+        can not undo this action. I have warned you.
+      </p>
+    </Modal.Body>
+    <Modal.Footer>
+      <Button variant="success" onClick={() => handleDelete(postId)}>Delete</Button>
+      <Button variant="danger" onClick={closeConfirmModal}>Close</Button>
```

Comment:
Tady bych to trochu pozmenil:
 - prvni button bych dal 'cancel' na close modal - ktery nebude bude barevne neutralni
 - druhy button 'delete' nebo 'confirm delete' - ktery bude cerveny (danger)